Source/Resources/Content/MSGraph/DeviceManagement/DeviceConfigurations/Security - NTLMv2 LAN Manager Authentication Level.json

```diff
@@ -8,6 +8,6 @@
       }
     }
   ],
-  "description": "Configures the LAN Manager Authentication Level to NTLMv2.",
+  "description": "What is it: Configures the LAN Manager Authentication Level to NTLMv2. \n Learn More: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/network-security-lan-manager-authentication-level",
```

Comment:
**What does this do?**
Configures the Windows LAN Manager Authentication Level to require NTLMv2. 

**Why should you use this?**
This is recommended by Microsoft to prevent the use of insecure protocols.  

**What is the end-user impact?**
Windows file sharing will not work with any devices that to not support NTLMv2.

**Learn more**
[Network security: LAN Manager authentication level](https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/network-security-lan-manager-authentication-level)